content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/README.md

```diff
@@ -79,6 +79,7 @@ The following properties are written to JCR for this Page component and are expe
 media
 20. `./cq:contextHubPath` - defines the Context Path configuration used by this page.
 21. `./cq:contextHubSegmentsPath` - defines the Context Path Segments Path.
+22. `./mainContentSelector` - defines selector for main content element of the page to be used by accessibility feature "skip to main content".
```

Comment:
I'd rename as: defines the selector for the main content element of the page (used by the "skip to main content" accessibility feature).